services/src/main/java/io/scalecube/services/Microservices.java

```diff
@@ -202,12 +200,33 @@ public Builder seeds(Address... seeds) {
      * @return builder.
      */
     public Builder services(Object... services) {
+
       checkNotNull(services);
       if (services != null) {
-        this.services = services;
+        this.servicesConfig = ServicesConfig.builder(this)
+            .services(services)
+            .create();
+      }
+      return this;
+    }
+
+    /**
+     * Services list to be registered.
+     * 
+     * @param servicesConfig list of instances decorated with.
+     * @return builder.
+     */
+    public Builder services(ServicesConfig servicesConfig) {
+      checkNotNull(servicesConfig);
+      if (servicesConfig != null) {
+        this.servicesConfig = servicesConfig;
```

Comment:
it seems that this method is doing nothing since servicesConfig is inited with `ServicesConfig.empty()` and basically never is null.